percona/controller/pgcluster/controller.go

```diff
@@ -152,6 +155,17 @@ func (r *PGClusterReconciler) Reconcile(ctx context.Context, request reconcile.R
 		return ctrl.Result{}, err
 	}
 
+	if cr.Spec.Pause != nil && *cr.Spec.Pause {
+		backupRunning, err := isBackupRunning(ctx, r.Client, cr)
+		if err != nil {
+			return reconcile.Result{}, errors.Wrap(err, "is backup running")
+		}
+		if backupRunning {
+			*cr.Spec.Pause = false
+			log.Info("Backup is running. Can't pause cluster", "cluster", cr.Name)
```

Comment:
Could we add here something more obvious to the user? If the user pauses the cluster and we again set it to false because of the running backup, it can produce some confusion since he will probably miss this log message. 

Maybe we can pause it after backup is finished? Would this be too much? Afterall chances for this case to happen is rather small.

@hors @egegunes wdyt?

Reply:
I think we need to pause the cluster as soon as the backup is finished. @egegunes wdyt?

Reply:
it'd be nice, yes

Reply:
We set it to false, but we don't update the cr resource. This means, that the operator will pause the cluster as soon as the backup is the finished